committee_history.md

```diff
@@ -117,7 +117,7 @@
   - Tom Cranitch
   - Darren Fu
   - Madhav Kumar
-  - Matt Low
+  - Matthew Low
   - Olivia MacKenzie-Ross
```

Comment:
For the sake of pedantry, I think this should also just be Mackenzie, not MacKenzie.

Reply:
Agreed 👍  Fixed